.github/workflows/docker_test.yml

```diff
@@ -35,7 +35,7 @@ jobs:
       - name: Test image
         run: |
           expected_version=$(cat pyproject.toml | grep version | head -n 1 | cut -d '"' -f 2)
-          version=$(docker run --rm --entrypoint bash langflowai/langflow:latest-dev -c 'python -c "from langflow.version import __version__ as langflow_version; print(langflow_version)"')
+          version=$(docker run --rm --entrypoint bash langflowai/langflow:latest-dev -c 'python -c "from langflow.utils.version import get_version; print(get_version())"')
```

Comment:
Hm that's weird. `get_version` is from here, no? https://github.com/langflow-ai/langflow/blob/main/src/backend/langflow/version/version.py#L6

Should it be this? https://github.com/langflow-ai/langflow/blob/main/src/backend/base/langflow/utils/version.py#L54

Reply:
(or `get_version_info()`)

Reply:
Good eyes!